dashboard/sample_metadata_classifier.py

```diff
@@ -293,11 +294,19 @@ def interpret(project, papers, bits):
                 country="Germany",
                 city="Heidelberg",
                 collection="plasma")
-    elif project in papers["Cebria-Mendoza 2021"]["projects"]:
-        sample, = bits
+    #elif project in papers["Cebria-Mendoza 2021"]["projects"]:
+    #    sample, = bits
+    #    return sample, dict(
+    #            country="Spain",
+    #            city="Valencia",
+    #            collection="plasma")
```

Comment:
Delete instead of commenting out.  We can always restore from version control.

Reply:
done